mmv1/templates/terraform/examples/dataproc_metastore_service_basic.tf.erb

```diff
@@ -0,0 +1,11 @@
+resource "google_dataproc_metastore_service" "<%= ctx[:primary_resource_id] %>" {
+  provider   = google-beta
+  service_id = "<%= ctx[:vars]['metastore_service_name'] %>"
+  location   = "us-central1"
+  port = 9083
```

Comment:
Can you change this to set a non-default value just to make sure it all works correctly?

Reply:
sure !